test/playwright/playwright.config.ts

```diff
@@ -1,7 +1,15 @@
+import path from 'path'
+
+import { addAliases } from 'module-alias'
+
 import type { PlaywrightTestConfig } from '@playwright/test'
 
+addAliases({
+  '~': path.resolve(process.cwd(), 'src'),
+  '~~': process.cwd(),
+})
+
 const config: PlaywrightTestConfig = {
-  testDir: '.',
```

Comment:
By default playwright recursively checks the directory that the configuration file is in, so this setting was redundant with the default behavior.